src/navier_stokes/INSVCStaggeredPressureBcCoef.cpp

```diff
@@ -327,9 +327,14 @@ INSVCStaggeredPressureBcCoef::setBcCoefs(Pointer<ArrayData<NDIM, double> >& acoe
                     }
                     else if (d_mu_interp_type == VC_HARMONIC_INTERP)
                     {
-                        mu = mu_contains_cells ?
-                                 2.0 * ((*mu_data)(i_g) * (*mu_data)(i_i)) / ((*mu_data)(i_g) + (*mu_data)(i_i)) :
-                                 -1.0e305;
+                        double mu_harmonic;
+                        if (IBTK::rel_equal_eps((*mu_data)(i_g), 0.0) || IBTK::rel_equal_eps((*mu_data)(i_i), 0.0))
```

Comment:
You should be very careful about using `rel_equal_eps` to compare numbers that are close to zero. I think you want to use `abs_equal_eps`. See these tests
https://github.com/IBAMR/IBAMR/blob/master/tests/IBTK/equal_eps.output

Reply:
Thanks @abarret. We switched to `abs_equal_eps`. Any idea what would be its equivalent function in fortran? We are using the `EPSILON()` function, i.e., `EPSILON(0.d0)`.

Reply:
> https://github.com/IBAMR/IBAMR/blob/master/tests/IBTK/equal_eps.output

https://github.com/IBAMR/IBAMR/blob/master/tests/IBTK/equal_eps.output#L65-L67

@abarret  Why are a = 1e-8 and b = 0 equal in the absolute test?

Reply:
similarly I don't understand this equality either
https://github.com/IBAMR/IBAMR/blob/master/tests/IBTK/equal_eps.output#L77-L79

Reply:
The equivalent check in fortran to `IBTK::abs_equal_eps` with the default tolerance would be
```
eps = EPSILON(0.d0)
abs_equal_eps = ABS(a - b) .LE. SQRT(eps)
```

Reply:
> The equivalent check in fortran to `IBTK::abs_equal_eps` with the default tolerance would be
> 
> ```
> eps = EPSILON(0.d0)
> abs_equal_eps = ABS(a - b) .LE. SQRT(eps)
> ```

Thanks!